about-page.html

```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE html>
+<html lang="en-us">
+
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Go Green - Summit 2022</title>
+    <meta name="description" content="About Go Green - Summit 2022">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="stylesheets/styles.css">
+</head>
+
+<body class="about">
+    <section class="top-section">
+        <div class="top-cover">
+            <nav class="inactive">
+                <a href="#close"><img src="images/bars.png" alt="Nav Button"></a>
+                <a href="index.html" class="logo"><img src="images/green_logo.png" alt="logo"></a>
+                <ul>
+                    <li class="Home">
+                        <a href="index.html">Home</a>
+                    </li>
+
+                    <li>
+                        <a href="about-page.html" class="active">About</a>
+                    </li>
+                    
+                    <li>
+                        <a href="#">Program</a>
+                    </li>
+                    
+                    <li>
+                        <a href="#">Join</a>
+                    </li>
+                    
+                    <li>
+                        <a href="#">Sponser</a>
+                    </li>
+                    
+                    <li>
+                        <a href="#">News</a>
+                    </li>
+                    
+                    <li>
+                        <a href="#">CC Compaign</a>
+                    </li>
+                </ul>
+            </nav>
+            <div class="content">
+                <header>
+                    <p>"Hello! Sharing world"</p>
+                    <h1>GoGreen Summit 2022</h1>
+                </header>
+                <p>
+                    Lorem ipsum dolor sit amet consectetur adipisicing elit. Voluptas earum quia eos ad modi consequuntur! Reprehenderit cupiditate, maxime.
+                </p>
+                <div class="top-section-desc">
+                    <p>Please contact us per Email for any further questions about GoGreen Summit 2022</p>
+                    <a href="mailto:saadatali0202@gmail.com">gogreen@gmail.com</a>
+                </div>
+            </div>
+        </div>
+    </section>
+
+    <section class="conference-logo-section">
+        <h3 class="section-header">
+            Creative Commons Global GoGreen Summit 2022
+            <span class="underline"></span>
```

Comment:
- [ ] Hello the first and last sections look good. However, I'd like to point out that you are actually missing the middle section of the about page that should be between these two sections. 
- [ ] This middle section should contain a centered column that includes a section header, a divider, a paragraph of text, and an image or large logo as the image below shows. please implement this change.

### The missing section in the Reference Design
![image](https://user-images.githubusercontent.com/18730106/152666300-ebd936f2-cd8b-41e5-b4a6-ca53171a3f40.png)